nntrainer/layers/common_properties.h

```diff
@@ -1075,6 +1075,19 @@ class BNPARAMS_BETA_INIT final : public EnumProperty<InitializerInfo> {
   static constexpr const char *key = "beta_initializer";
 };
 
+/**
+ * @brief RMS_NORM_GAMMA_INIT Initialization Enumeration Information
+ */
+class RMS_NORM_GAMMA_INIT final : public EnumProperty<InitializerInfo> {
```

Comment:
It is quite strange to use capital letters here.

Reply:
Oh I checked BNPPARAMS_*_INIT only.  
As you suggested, it seems more natural to update it to follow our convention :) I will apply it